src/comments.js

```diff
@@ -369,6 +372,22 @@ function handleMemberExpressionComment(enclosingNode, followingNode, comment) {
   return false;
 }
 
+function handleTemplateLiteralComment(enclosingNode, comment, isOwnLine) {
+  if (
+    enclosingNode &&
+      enclosingNode.type === "TemplateLiteral"
+  ) {
+    if (isOwnLine) {
+      addTrailingComment(enclosingNode.expressions[0], comment);
```

Comment:
what happens if there's more than one expression? Is it going to pick the right one?